src/main/java/com/openmrs/migrator/core/services/impl/PDIServiceImpl.java

```diff
@@ -45,4 +57,19 @@ public void runTransformation(InputStream xmlStream) throws KettleException {
 
     LOG.info(outcome);
   }
+
+  @Override
+  public void mergeOpenMRS() {
+    try {
+
+      for (String t : transformations) {
```

Comment:
So I'd like to see this managed from a PDI Job instead of here in the app. The PDI service should be exclusively focused on running PDI jobs which will in term manage the pipeline for running transformations.

let me know if this needs more explanation and we can setup a call to discuss

Reply:
I don't understand the term: managed from a PDI Job instead of here in the app.
We should setup a call.

Reply:
Ok now  understand the approach

Reply:
done